src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -20,273 +20,692 @@ PICS:
+                        GroupKeySetID: 0x01a1,
+                        GroupKeySecurityPolicy: 1,
+                        EpochKey0: "\xa0\xa1\xa2\xa3\xa4\xa5\xa6\xa7\xa8\xa9\xaa\xab\xac\xad\xae\xaf",
+                        EpochStartTime0: 0,
+                        EpochKey1: "\xb0\xb1\xb2\xb3\xb4\xb5\xb6\xb7\xb8\xb9\xba\xbb\xbc\xbd\xbe\xbf",
+                        EpochStartTime1: 0,
+                        EpochKey2: "\xc0\xc1\xc2\xc3\xc4\xc5\xc6\xc7\xc8\xc9\xca\xcb\xcc\xcd\xce\xcf",
+                        EpochStartTime2: 0,
+                    }
+
+    - label:
+          "Preparation step for using commands from Groups cluster: Write Group
+          Keys"
+      cluster: "Group Key Management"
+      endpoint: 0
+      command: "writeAttribute"
+      attribute: "GroupKeyMap"
+      arguments:
+          value:
+              [
+                  { FabricIndex: 1, GroupId: 0x0001, GroupKeySetID: 0x01a1 },
+                  { FabricIndex: 1, GroupId: 0x0002, GroupKeySetID: 0x01a1 },
+              ]
 
     - label:
           "TH sends a AddGroup command to DUT with the GroupID field set to
           0x0001."
       PICS: G.S.C00.Rsp
-      verification: |
-          DUT sends a AddGroupResponse command to TH with the Status field set to 0x00 (SUCCESS) and the GroupID field set to 0x0001.
-      disabled: true
+      cluster: "Groups"
+      command: "AddGroup"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+              - name: "GroupName"
+                value: "Group1"
+      response:
+          values:
+              - name: "Status"
+                value: 0
+              - name: "GroupID"
+                value: 0x0001
 
     - label:
           "TH sends a RemoveAllScenes command to DUT with the GroupID field set
           to 0x0001."
       PICS: S.S.C03.Rsp
-      verification: |
-          DUT sends a RemoveAllScenesResponse command to TH with the Status field set to 0x00 (SUCCESS) and GroupID field set to 0x0001.
-      disabled: true
+      command: "RemoveAllScenes"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+      response:
+          values:
+              - name: "Status"
+                value: 0
 
     - label:
           "TH sends a GetSceneMembership command to DUT with the GroupID field
           set to 0x0001."
       PICS: S.S.C06.Rsp
-      verification: |
-          DUT sends a GetSceneMembershipResponse command to TH with the Status field set to 0x00 (SUCCESS), the Capacity field set to the value SC0, the GroupID field set to 0x0001 and the SceneCount field set to 0x00.
-      disabled: true
+      command: "GetSceneMembership"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0001
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "Capacity"
+                value: 8
```

Comment:
Where is this 8 coming from?  This is going to be device-specific, no?  It needs to be something we either read from the device or pass into this script as an argument.